volontulo/templates/common/footer.html

```diff
@@ -26,6 +26,7 @@
             <h3>Oferty</h3>
             <ul>
                 <li><a href="{% url 'offer_apply' 1 %}">Zgłoś chęć współpracy przy ofercie</a></li>
+                <li><a href="{% url 'show_offer' 1 %}">Podgląd oferty</a></li>
```

Comment:
yes, these placeholders were meant for people like Dominik so they have skeleton of the navigation. 
It's no big deal at all, but it can help first-timers realise the site structure.